src/vivarium/framework/artifact/artifact.py

```diff
@@ -193,16 +193,16 @@ def clear_cache(self) -> None:
         """
         self._cache = {}
 
-    def __iter__(self):
+    def __iter__(self) -> Iterable[str]:
```

Comment:
Should return type `Iterator`

Reply:
You have to provide the type arg too.

Reply:
I mean `__iter__(self) -> Iterator[str]:`